src/nrniv/secbrows.cpp

```diff
@@ -390,8 +389,6 @@ SectionBrowserImpl::~SectionBrowserImpl() {
 
 SectionBrowser::SectionBrowser()
     : OcBrowser(new BrowserAccept(this), NULL) {
-    LayoutKit& lk = *LayoutKit::instance();
-    WidgetKit& wk = *WidgetKit::instance();
```

Comment:
Very risky, because they allocate a global. Therefore, if unlucky this affects the order in which those doubles are deallocated, potentially causing a bug. I'd suggest either a separate PR or using `[[maybe_unused]]`.